### ISSUE TITLE
audiodg appears as a program in the volume mixer interface

### ISSUE THREAD

**Issue by migmarquez** [File-New-Project/EarTrumpet#1095]
### Summary

when we open EarTrumpet from the taskbar audiodg appears as a program in the volume mixer interface and it should not appear as it is a windows service.

### Steps to reproduce

Step 1: Click on EarTrump icon in taskbar

### EarTrumpet version

2.1.9.0

### Windows version

11

### Additional information

![image](https://user-images.githubusercontent.com/101899691/159070418-71a48a5c-c664-48c7-b20d-a33cde3834b2.png)

**Comment by migmarquez** [comment]
![image](https://user-images.githubusercontent.com/101899691/159070795-83f7119a-e6b0-4473-b232-f525c2f63b69.png)

**Comment by riverar** [comment]
Does this also show up in the Windows audio mixer? (Right-click EarTrumpet > Windows Legacy > Sound Settings > Volume Mixer)

**Comment by migmarquez** [comment]
> Does this also show up in the Windows audio mixer? (Right-click EarTrumpet > Windows Legacy > Sound Settings > Volume Mixer)

![image](https://user-images.githubusercontent.com/101899691/159072133-059514e7-189a-48b2-8799-68d2e4f16ced.png)
apparently discord is duplicated in EarTrumpet and the second discord recognizes it as audiodg

**Comment by riverar** [comment]
I was looking to see what happens in the volume mixer located in Settings. (Right-click EarTrumpet > Windows Legacy > Sound Settings > Volume Mixer)

**Comment by migmarquez** [comment]
> I was looking to see what happens in the volume mixer located in Settings. (Right-click EarTrumpet > Windows Legacy > Sound Settings > Volume Mixer)

![image](https://user-images.githubusercontent.com/101899691/159073300-7258cc15-134c-491e-aab3-5b3fdbe684db.png)

**Comment by riverar** [comment]
Interesting. Any ideas on how I can reproduce this here? Are you maybe in a Discord call? Or watching a live stream/video?

**Comment by migmarquez** [comment]
> Interesting. Any ideas on how I can reproduce this here? Are you maybe in a Discord call? Or watching a live stream/video?

![image](https://user-images.githubusercontent.com/101899691/159077124-36431932-b3b0-44a7-8870-0c51fe222110.png)
I have closed all my applications and it is still showing up, I am attaching a picture of all the mixers.
The weird thing about this is that after I noticed this I installed another program called Volume Control for Windows from the Microsoft Store based on the EarTrumpet source and that program does not show the audiodg in the mixer

**Comment by riverar** [comment]
Fun! Open EarTrumpet Settings > General > About > Collect troubleshooting data. That'll dump the sessions and we can take a closer look at that audiodg.

**Comment by migmarquez** [comment]
[tmp6F16.tmp.txt](https://github.com/File-New-Project/EarTrumpet/files/8307450/tmp6F16.tmp.txt)

**Comment by riverar** [comment]
Yeah its a component in Windows Audio; it appears we're the only app behaving correctly but I'll take a closer look on my end. These built-in apps may be filtering out noise like this and maybe we should too.

**Comment by migmarquez** [comment]
> Yeah its a component in Windows Audio; it appears we're the only app behaving correctly but I'll take a closer look on my end. These built-in apps may be filtering out noise like this and maybe we should too.

evidently maybe audiodg being a service that is linked to the audio driver and the EarTrumpet operating system recognizes it as a program that is emitting a sound, but the solution could be to prevent eartrumpet from filtering this audiodg.exe and not include it in the mixer.

**Comment by ThatOtherAndrew** [comment]
Can confirm that this is an issue still present for me as well. audiodg seems to be showing that it's playing back the same audio as Discord when in a call (i.e. the level bar moves in the same way, but with more latency), however muting it doesn't seem to do anything.

**Comment by riverar** [comment]
Ah great @ThatOtherAndrew, that's some information I was missing (how to reproduce it). I'll try this out tonight and if I reproduces, get a fix in.

**Comment by sinni800** [comment]
A way to reproduce is to install SteelSeries' GG software, which creates a handful of virtual audio devices meant to separate the sound out into chat, gaming, media and "aux" applications for mixing purposes (the SteelSeries headsets have chat-gaming balance sliders) and for the purpose of having different EQs on each of them. The software encourages you to use the audio device routing in Windows 11 to throw apps onto different of these output devices.

In the background what they do is make audiodg do the copying here (I think?). Weirdly in the Windows mixer it shows up as the SteelSeries icon with no name, but EarTrumpet shows it as an audiodg process that basically has the volume of the total output of the virtual audio device, playing it into the physical one. This shows it like this in the debug info you mentioned: 

```
--------------------------------------------------------------------
4 - Q85A (AMD High Definition Audio Device) 74% Id: {0.0.0.00000000}.{5c1bd195-22cf-4fb6-ba64-5a8892621a07}
--------------------------------------------------------------------
  audiodg
    (Desktop) (Active) 100% Id: {0.0.0.00000000}.{5c1bd195-22cf-4fb6-ba64-5a8892621a07}|#%b{1E8A0200-A28B-41C0-9BAF-328F7BEF863A}|0%b#
    AppId: audiodg.exe ProcessId: 15704 Alive: True
    IconPath: audiodg.exe
    GroupingParam: 7f384aa0-ba7d-4ef3-8bd8-a0a7c81c8381
  audiodg
    (Desktop) (Active) 100% Id: {0.0.0.00000000}.{5c1bd195-22cf-4fb6-ba64-5a8892621a07}|#%b{1E8A0500-A28B-41C0-9BAF-328F7BEF863A}|0%b#
    AppId: audiodg.exe ProcessId: 15704 Alive: True
    IconPath: audiodg.exe
    GroupingParam: 7e16678d-30e1-47ef-8550-48746cb1be3a
```

The Q85A is my current physical device (chosen in the GG software) outputting the sound. 

This also made me realize how poor Discord is in how it just registers itself to EVERY audio device as a stream. Gosh, if every app did this...
